neard/res/mainnet_genesis.json

```diff
@@ -0,0 +1,336 @@
+{
+  "config_version": 1,
+  "protocol_version": 7,
+  "genesis_time": "2020-04-22T00:00:00.000000000Z",
+  "chain_id": "mainnet",
+  "genesis_height": 0,
+  "num_block_producer_seats": 100,
+  "num_block_producer_seats_per_shard": [
+    100
+  ],
+  "avg_hidden_validator_seats_per_shard": [
+    0
+  ],
+  "dynamic_resharding": false,
+  "epoch_length": 43200,
+  "gas_limit": 1000000000000000,
+  "min_gas_price": "100000",
+  "block_producer_kickout_threshold": 90,
+  "chunk_producer_kickout_threshold": 60,
+  "gas_price_adjustment_rate": 1,
+  "runtime_config": {
+    "storage_amount_per_byte": "90900000000000000000",
```

Comment:
I don't think it would be a huge difference if we round it up to `100000000000000000000`. However, it will make accounting easier.

Reply:
Well, it's either way weird if you count in KB:

Storage | N locked for Storage | % of Total Supply
-- | -- | --
10 MB | 954 | 0.0001%
100 MB | 9,537 | 0.0010%
1 GB | 97,656 | 0.0098%
10 GB | 976,563 | 0.098%
100 GB | 9,765,625 | 0.98%
1 TB | 100,000,000 | 10.00%

or
Storage | N locked for Storage | % of Total Supply
-- | -- | --
10 MB | 105 | 0.0000%
100 MB | 1,049 | 0.0001%
1 GB | 10,737 | 0.0011%
10 GB | 107,374 | 0.011%
100 GB | 1,073,742 | 0.11%
1 TB | 10,995,116 | 1.10%

But I don't mind rounding it up.

Reply:
Let's round it up, people will be transferring rounded number of tokens to their accounts based on the contract size, which is nice. E.g. I am as a developer of of 163k contract will immediately know that 1.63N is enough.